WORKFLOW.md

```diff
@@ -0,0 +1,129 @@
+# Development workflows
+
+This document prescribes how to branch and merge code in this project.
+
+When provided, shell commands are designed to minimize human interaction.
+In most cases the commands can be copied and pasted.
+
+Some commands may not be available until the virtual environment is created and activated.
+
+## Table of contents
+
+* [Priority git branches](#priority-git-branches)
+* [Everyday development](#everyday-development)
+* [Preparing a feature release](#preparing-a-feature-release)
+* [Preparing a hotfix release](#preparing-a-hotfix-release)
+* [Merging release branches](#merging-release-branches)
+* [Deploying to production](#deploying-to-production)
+
+## Priority git branches
+
+There are several git branches that have assigned significance.
+
+### `main`
+
+`main` tracks all repository changes. Every branch must eventually merge to 
+`main`.
+
+### `production`
+
+`production` tracks code that is, or is about to be, deployed to PyPI or
+ReadTheDocs. Each merge commit must be accompanied by a git tag representing
+the released version.
+
+## Everyday development
+
+"Everyday" development refers to repository changes that are not intended for  
+out-of-cycle release to the production environment. This may include 
+non-critical bug fixes, documentation updates, CI/CD changes, dependency 
+updates, or other tooling changes.
+
+Feature development begins by creating a new branch off of `main`:
```

Comment:
It may be useful to copy in the text regarding Shortcut integration from the flows repo's workflow doc.

Reply:
Done!